rosetta/configuration/version.go

```diff
@@ -15,7 +15,7 @@
 package configuration
 
 const (
-	RosettaVersion    = "1.4.10"
-	NodeVersion       = "1.17.4"
-	MiddlewareVersion = "0.0.0"
+	RosettaVersion    = "v1.4.10"
```

Comment:
Rosetta does not explicitly say it shouldn't be there, but in the docs they do not use the `v` prefix in the version string - https://www.rosetta-api.org/docs/models/Version.html